esignet-insurance.properties

```diff
@@ -469,8 +469,36 @@ mosip.esignet.vci.key-values={\
                                   'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'},\
                                   'background_color': '#FDFAF9',\
                                     'text_color': '#7C4616'}}\
-                 }}\
-        }\
+                 },\
+                  "LifeInsuranceCredential":{\
```

Comment:
We need to update this under v11 also based on draft 11 spec and this section should be based on draft 12 spec.

Reply:
added under v11